accelerator/http.c

```diff
@@ -214,7 +212,7 @@ static int ta_http_handler(void *cls, struct MHD_Connection *connection,
                             "application/json");
   }
 
-  ret = MHD_queue_response(connection, MHD_HTTP_OK, response);
+  ret = MHD_queue_response(connection, ret, response);
```

Comment:
I didn't understand the motivation of this change.

Reply:
The second parameter is HTTP status code, HTTP status code `ret` is returned from `ta_http_request_process`. It should not always be `200 OK`.